src/Symfony/Bridge/Twig/Tests/Extension/LinkExtensionTest.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Twig\Tests\Extension;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Bridge\Twig\Extension\LinkExtension;
+use Symfony\Component\Link\LinkManager;
+
+/**
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+class LinkExtensionTest extends TestCase
+{
+    protected function setUp()
+    {
+        if (!class_exists(LinkManager::class)) {
```

Comment:
I would add `symfony/link` to the `require-dev` section of the `composer.json` file instead.